client/packages/common/src/ui/components/inputs/CurrencyInput/CurrencyInput.tsx

```diff
@@ -37,11 +37,13 @@ export const CurrencyInput: FC<CurrencyInputProps> = ({
   defaultValue = 0,
   onChangeNumber,
   maxWidth,
+  value,
   ...restOfProps
 }) => {
   const { c, options, language } = useCurrency();
   const prefix = language !== 'fr' ? options.symbol : '';
   const suffix = language === 'fr' ? options.symbol : '';
+  const [rawValue, setRawValue] = React.useState(value);
```

Comment:
We'd normally include this in the import: `import React, { FC, useState } from 'react';`

And this would just be: `const [rawValue, setRawValue] = useState(value);`